pkg/services/oracle/neofs/neofs.go

```diff
@@ -44,17 +44,11 @@ var (
 // Get returns a neofs object from the provided url.
 // URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
 // If Command is not provided, full object is requested.
-func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) (io.ReadCloser, error) {
+func Get(ctx context.Context, c *client.Client, priv *keys.PrivateKey, u *url.URL, addr string) (io.ReadCloser, error) {
```

Comment:
May be just create some `GetWithClient` in addition to `Get`? To keep `github.com/nspcc-dev/neofs-sdk-go/client` dependency isolated in a `neofs` package.